src/target_entry.rs

```diff
@@ -0,0 +1,104 @@
+use ffi;
+use glib;
+use glib::translate::*;
+use gobject_ffi;
+use std::mem;
+use TargetFlags;
+
+#[derive(Clone, Debug)]
+#[repr(C)]
+pub struct TargetEntry {
+    pub target: String,
```

Comment:
Is there a reason these shouldn't be public? All manually implemented structs so far have used public fields.